webpack.common.config.ts

```diff
@@ -402,18 +402,6 @@ module.exports = {
       minify: htmlMinifyConfig,
       inject: false,
     }),
-    new HtmlWebpackPlugin({
```

Comment:
We need to remove `topic_viewer` in other parts of this file. (See other merged PRs for #19435 for reference)

Reply:
Made the necessary changes